src/components/webViewRe.re

```diff
@@ -0,0 +1,289 @@
+module type WebViewComponent = {
+  type webViewNavState;
+  let webViewNavState:
+    url::string? =>
+    title::string? =>
+    loading::bool? =>
+    canGoBack::bool? =>
+    canGoForward::bool? =>
+    unit =>
+    webViewNavState;
+  type webViewIOSLoadRequestEvent;
+  let webViewIOSLoadRequestEvent:
+    target::int =>
+    canGoBack::bool =>
+    lockIdentifier::int =>
+    loading::bool =>
+    title::string =>
+    canGoForward::bool =>
+    navigationType::[ | `other | `click] =>
+    url::string =>
+    unit =>
+    webViewIOSLoadRequestEvent;
+  type webViewSource;
+  let webViewSource:
```

Comment:
I think `source` is enough since you would access it with `WebView.source`, so `WebView.webViewSouce` would be kinda redundant